src/openms/source/FORMAT/OMSFileLoad.cpp

```diff
@@ -41,8 +41,12 @@
 #include <QtSql/QSqlDatabase>
 #include <QtSql/QSqlError>
 #include <QtSql/QSqlQuery>
+#include <QtSql/QSqlRecord>
 // strangely, this is needed for type conversions in "QSqlQuery::bindValue":
 #include <QtSql/QSqlQueryModel>
+// JSON export:
```

Comment:
can we go with some additional json dependency - we already use the nlohmann json library.

Reply:
Can you elaborate?

Reply:
#include <QtCore/QJsonArray> // for JSON export
We already have https://github.com/OpenMS/OpenMS/blob/develop/src/openms/thirdparty/json
and I would love to not have QJson here. Especially as I would love to see the use of Qt reduced in the core library (e.g. GUI would be a different story).